scripts/benchmarks/benches/compute.py

```diff
@@ -355,3 +375,38 @@ def bin_args(self) -> list[str]:
             f"--SrcUSM={self.srcUSM}",
             f"--DstUSM={self.dstUSM}",
         ]
+
+class GraphApiSinKernelSYCL(ComputeBenchmark):
+    def __init__(self, bench, graphs, numKernels):
+        self.graphs = graphs
+        self.numKernels = numKernels
+        super().__init__(bench, "graph_api_benchmark_sycl", "SinKernel")
+
+    def name(self):
+        return f"graph_api_benchmark_sycl SinKernel"
```

Comment:
names need to reflect arguments, otherwise the benchmarks won't be unique in output.